node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -309,7 +308,8 @@ class NodeRuntime private[node] (
       port,
       conf.tls.certificate,
       conf.tls.key,
-      conf.server.maxMessageSize
+      conf.server.maxMessageSize,
+      conf.server.dataDir.resolve("comm-temporary/")
```

Comment:
maybe it's a good opportunity to introduce a naming scheme?
e.g.
`tmp/comm`?

Reply:
👍